proposals/lip-0048.md

```diff
@@ -282,11 +280,11 @@ def afterCommandExecute(trs: Transaction) -> None:
 #### Before Cross-chain Command Execution
 
 ```python
-def beforeCrossChainCommandExecution(trs: Transaction, ccm: CCM) -> None:
+def beforeCrossChainCommandExecution(ccu: Transaction, ccm: CCM) -> None:
```

Comment:
This is fine as well, but I find this to be a cosmetic change at this point.

Reply:
I updated the PR and now use universally the variable `ccu` for type `CCU` and `trs` for type `Transaction`.